### PR TITLE
Add warning to account creation email.

### PR DESCRIPTION
**Pull request by mountHouli**
# Related Tickets

- [ORANGE-1170](https://jira.amida.com/browse/ORANGE-1170)

# Other Repos' PR(s) Intended to Work With This PR

None.

# How Things Worked (or Didn't) Before This PR

The account creation email said:

> Hi {{user.firstName}},
>
> Welcome to MLA Connect!
>
> -- The MLA Team

# How Things Work Now (And How to Test)

Now it says:

> Hi {{user.firstName}},
>
> Welcome to MLA Connect!
>
> -- The MLA Team
>
> If you feel this email has been sent to you in error, please contact support@mlaconnect.zendesk.com

Also, I updated things to work with sendgrid v5.

**To Test**:

1. (developers) Setup `orange-api` and `orange-web` to run locally.
2. In `orange-web` create a user with `yourRealEmailAddress+something@amida.com`.
3. You should get a welcome email with this content.

# Readiness

1. [x] This PR has full test coverage (If this PR fixes a bug, you _must_ add test cases respresentative of the bug).
2. [x] This PR passes all automated tests.
3. [x] This PR has no linting errors.
4. [x] This PR has no hardcoded UI strings or other obvious issues.
5. [x] This PR has been updated to include all changes from develop.
6. [x] This PR's changes to configuration files have been documented in all appropriate places (such as but not limited to README.md), if applicable.
7. [x] This PR's required changes outside of the scope of this repository have been documented in this pull request.

### CODE REVIEW

File: lib/models/user/notifications.js [hunk 1]
```diff
@@ -133,11 +138,25 @@ module.exports = function (UserSchema) {
...
+            });
+
             // function to send an email notification
             var sendEmail = function () {
-                mailer.send(messages.email, function (err) {
+                mailer.API(sgRequest, function (err) {
                     // errors are SILENTLY logged and ignored
-                    if (err) console.error(err);
+                    if (err) {
+                        console.log("Error sending email: ", err);
```
Comment (dbrezack) [inline lib/models/user/notifications.js]: Is there a better option than `console.log` ?

Comment (mountHouli) [reply lib/models/user/notifications.js]: Good question; I thought the same thing. I checked and the Amida logger isn't implemented in this repo, and `console.log` is used in other places, which is why I used it.

I wish I would have discovered this sooner. We could have implemented the Amida logger in this repo.

Comment (mountHouli) [reply lib/models/user/notifications.js]: I'll make a ticket for that--though it won't be getting in at this time.

**Comment by hmayenve** [comment]
Tested on dev and it failed.

**Pull request merged**